### DISCUSSION TITLE
Lot of emails that remain in the queue

### DISCUSSION THREAD

**Discussion by SaasX** [postalserver/postal#2359] [Help with using Postal]
Hello,

I use postal on a dedicated server for multiple clients. The long-term idea is to bring together 200 users who send small newsletters.

I have an ip pool with dedicated addresses for each clients.

I have a few questions because I have a lot of emails that remain in the queue: they don't send OR take a very long time.

My server configuration:
Intel Xeon-E 2136 - 6c/12t - 3.3GHz/4.5GHz
32GB DDR4 ECC 2666MHz
2× 500GB NVMe Soft RAID SSD

If I look at the performance of the server, it's not running out of memory or CPU, so i don't understand what is the problem.

Do you have any ideas to improve this point or the causes of this concern?

Regards.

**Comment by SaasX** [comment]
I would add that the problems are more frequent on certain mail servers, but not on others, even though the parameters are the same.

**Reply by SaasX** [reply]
Reading the forum topics, the solution would probably be to add workers / threads

I added in postal.yml:
```
workers:
  quantity: 10
  threads: 10
```
I restarted postal but I don't see any changes with postal status
The subjects date from version 1 of postal, for version 2 how to do?

**Reply by willpower232** [reply]
When running Postal v2 in docker-compose, you will need to use the scale option I believe to add more workers. The quantity setting doesn't have an effect any more, I'm not sure about the threads one.

**Reply by SaasX** [reply]
thank you for your reply.
This is a point that I do not master at all :S
Do you have a link to documentation or a tutorial?

**Reply by SaasX** [reply]
A question regarding postal 2, is there a worker for all smtp or a worker per smtp?

**Reply by slobodangacesa** [reply]
> thank you for your reply. This is a point that I do not master at all :S Do you have a link to documentation or a tutorial?

postal start --scale worker=10

**Reply by SaasX** [reply]
So simple it's magic ;) 
Thanks a lot
